.github/workflows/autotest.yml

```
@@ -21,6 +22,18 @@ jobs:
       - name: Run hatch test
         shell: bash
         run: hatch test
-      - name: Code Formatting check
+  formattest:
+    name: Code formatting check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v5
```

```suggestion
      - name: Install Python
        uses: actions/setup-python@v5
```
(every other step has  a name)